python/run-tests.py

```diff
@@ -20,48 +20,84 @@
 import fnmatch
 import subprocess
 import sys
+import shutil
 from os import path
 
 
-def test(root_dir, jar_file):
-    # And then, run all of the test under test/python directory, each of them
+def test(root_dir):
+    # Run all of the test under test/python directory, each of them
     # has main entry point to execute, which is python's unittest testing
     # framework.
-    test_dir = path.join(root_dir,
-                         path.join("python", path.join("delta", "tests")))
+    python_root_dir = path.join(root_dir, "python")
+    test_dir = path.join(python_root_dir, path.join("delta", "tests"))
     test_files = [os.path.join(test_dir, f) for f in os.listdir(test_dir)
                   if os.path.isfile(os.path.join(test_dir, f)) and
                   f.endswith(".py") and not f.startswith("_")]
+    extra_class_path = path.join(python_root_dir, path.join("delta", "testing"))
+    print(extra_class_path)
+
+    # Get Current release which is required to be loaded
+    version = '0.3.0'
+    try:
+        with open(os.path.join(root_dir, "version.sbt")) as fd:
+            version = fd.readline().split('"')[1]
+    except:
+        raise Exception("Could not find current release version" +
+                        "Please check version.sbt")
+
+    package = "io.delta:delta-core_2.11:" + version
```

Comment:
`test()` should get this package name detail from `prepare()` (like it used to get the jar file path before this change")